presto-verifier/src/main/java/com/facebook/presto/verifier/framework/QueryRewriter.java

```diff
@@ -38,29 +40,36 @@
 import java.util.Map;
 import java.util.Optional;
 import java.util.Set;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
 
 import static com.facebook.presto.sql.tree.LikeClause.PropertiesOption.INCLUDING;
 import static com.facebook.presto.verifier.framework.QueryOrigin.TargetCluster.CONTROL;
 import static com.facebook.presto.verifier.framework.QueryOrigin.TargetCluster.TEST;
 import static com.facebook.presto.verifier.framework.QueryOrigin.forRewrite;
 import static com.facebook.presto.verifier.framework.QueryType.Category.DATA_PRODUCING;
 import static com.facebook.presto.verifier.framework.VerifierUtil.PARSING_OPTIONS;
+import static com.google.common.collect.ImmutableList.toImmutableList;
+import static com.google.common.collect.ImmutableMap.toImmutableMap;
 import static java.lang.String.format;
 import static java.util.Locale.ENGLISH;
+import static java.util.Map.Entry;
 import static java.util.Objects.requireNonNull;
 import static java.util.UUID.randomUUID;
 
 public class QueryRewriter
 {
     private final SqlParser sqlParser;
     private final PrestoAction prestoAction;
+    private final List<Property> tablePropertiesOverride;
     private final Map<TargetCluster, QualifiedName> prefixes;
 
     @Inject
-    public QueryRewriter(SqlParser sqlParser, PrestoAction prestoAction, VerifierConfig config)
+    public QueryRewriter(SqlParser sqlParser, PrestoAction prestoAction, List<Property> tablePropertiesOverride, VerifierConfig config)
```

Comment:
nit: From english perspective, should this be `tablePropertyOverrides` or `tablePropertiesOverride` ? 

cc @rschlussel

Reply:
I think the first